OrbitCore/CaptureData.h

```diff
@@ -132,6 +132,15 @@ class CaptureData {
 
   [[nodiscard]] const CallstackData* GetCallstackData() const { return callstack_data_.get(); };
 
+  [[nodiscard]] Mutex& GetTracepointEventBufferMutex() const {
+    return tracepoint_event_buffer_->mutex();
+  }
+
+  [[nodiscard]] std::map<int32_t, std::map<uint64_t, orbit_client_protos::TracepointEventInfo> >
```

Comment:
This will make a copy of the map. I guess you want to return a (const?) reference.

Reply:
Yes, of course. Thank you!